01-rstudio-intro.Rmd

```diff
@@ -378,10 +408,26 @@ rm(list <- ls())
 >
 
 
+#### R Packages
+
+It is possible to add functions to R by writing a package, or by
+obtaining a package written by someone else. As of this writing, there
+are over 7,000 packages available on CRAN (the comprehensive R archive
+network). R and RStudio have functionality for managing packages:
+
+* You can see what packages are installed by typing
+  `installed.packages()`
+* You can install packages by typing `install.packages("packagename")`,
+  where `packagename` is the package name, in quotes.
+* You can update installed packages by typing `update.packages()`
+* You can remove a package with `remove.packages("packagename")`
+* You can make a package available for use with `library(packagename)`
+
+
```

Comment:
I like this description but as it stands it seems a bit theoretical for novices, I wonder if it just adds more cognitive load. I think it would be nicer with an example, my favourite 'toy' package is 'magic'.